### ISSUE TITLE
UX issue manage options for attributes

### ISSUE THREAD

**Issue by davorgit** [isotope/core#1361]
I have set new attribute with Options Manager selected and select field in the past, 
options are shown in Product Management -> Product below but 
now I just cannot find(forgot) where to manage(add/delete/modify) these options 
it would be nice to have a message/notification where to find them and a little yellow pencil beside them to edit them

and please can you tell me now where are they :)
PS: I don't want to use Options Wizard (deprecated) feature just because of this problem :)

**Comment by davorgit** [comment]
screen shot of what I see, thus options are defined somewhere for this attribute
![no-options-shown](https://cloud.githubusercontent.com/assets/1269067/6076409/697d167c-ade3-11e4-89cc-5a09130417e5.jpg)

**Comment by davorgit** [comment]
Product management edit attribute options feature request:
![attribute-options-edit-feature](https://cloud.githubusercontent.com/assets/1269067/6076521/cad5904c-ade4-11e4-8c8c-f0efc8382e3a.jpg)

**Comment by aschempp** [comment]
I guess you did not correctly install the DcaWizard extension.

**Comment by davorgit** [comment]
what is correct way by extension manager or by zip-install tool?

**Comment by davorgit** [comment]
Yes, I've updated all extensions thru Extension Manager and it worked.
Still feature I mentioned would be great to have.

Thank You Sir

**Comment by davorgit** [comment]
Can you also tell me please how to display BE selected option label like FE function <?php echo $this->generateAttribute('color'); ?> outputs value id?
i.e. similar to: this->generateAttributeLabel('color')

**Comment by aschempp** [comment]
I don't think there's a shortcut for that currently. Please refer to the community forums for support.

**Comment by davorgit** [comment]
Ok, I have extened Model/Product/Standard class and added:

```php
$objTemplate->generateAttributeLabel = function ($strAttribute, array $arrOptions = array()) use ($objProduct) {
    $objAttribute = $GLOBALS['TL_DCA']['tl_iso_product']['attributes'][$strAttribute];
    if (!($objAttribute instanceof IsotopeAttribute)) {
        throw new \InvalidArgumentException($strAttribute . ' is not a valid attribute');
    }
    $strQuery = "SELECT label FROM tl_iso_attribute_option WHERE id=? AND published=?";
    $objLabel = \Database::getInstance()->prepare($strQuery)->limit(1)->execute($objProduct->arrData[$strAttribute], 1);
    return $objLabel->row()['label'];
};
```

I know it's dirty but it works ..i wanted to add it to arrData with buildFindQuery, but data is pulled from products table with dynamic added attribute fields ..so it would be way to much work and I think JOIN isn't possible ..correct me if I'm wrong

**Comment by aschempp** [comment]
Maybe we could add that in a future version :-)

**Comment by davorgit** [comment]
Yes that would be nice. 
I have another problem when adding this (modified) code to extended class \Isotope\Model\ProductCollection\Cart for display on shopping-cart ..code actually work if product is already added to cart, but not if there is no product yet and cookie is not set yet
..got these error below, which is connected with Cart.php line 274 $objCart   = new static();

..Fatal error: Uncaught exception RuntimeException with message Isotope\Model\ProductCollection\Cart has no model type thrown in ....\system\modules\isotope\library\Isotope\Model\TypeAgent.php on line 55

this happens when I registerModelType:
\Isotope\Model\CartProductCollection::registerModelType('cart', 'IsotopeMyOwnNS\CartProductCollection');

so in further investigation I found problem with method getCart() in Isotope.php line:112 calling Cart::findForCurrentStore()

so anyway problem is new static() call

**Comment by davorgit** [comment]
here is the code for Cart.php or ProductCollection.php

```php
$objTemplate->generateAttributeLabel = function ($strAttribute, ProductCollectionItem $objItem, array $arrOptions = array()) {
    	
    		if (!$objItem->hasProduct()) {
    			return '';
    		}
    
    		$objAttribute = $GLOBALS['TL_DCA']['tl_iso_product']['attributes'][$strAttribute];
    		
    		if (!($objAttribute instanceof IsotopeAttribute)) {
    			throw new \InvalidArgumentException($strAttribute . ' is not a valid attribute');
    		}
    		$strQuery = "SELECT label FROM tl_iso_attribute_option WHERE id=? AND published=?";
    		$objLabel = \Database::getInstance()->prepare($strQuery)->limit(1)->execute($objItem->getProduct()->arrData[$strAttribute], 1);
    		 
    		return $objLabel->row()['label'];
    	};
```

**Comment by qzminski** [comment]
Is this issue (with ```generateAttributeLabel()``` method) still relevant? I have just added a multiple checkbox attribute and it displays the option labels (not values) in the frontend. From what I see the field options are already correctly added to the DCA.

**Comment by aschempp** [comment]
I think this can be closed, there's also new `raw` options in 2.3
